Sources/HermesAppDelegate.m

```diff
@@ -558,30 +604,11 @@ - (IBAction) updateStatusItem:(id)sender {
     
   } else if (PREF_KEY_BOOL(STATUS_BAR_ICON_ALBUM)) {
     
-    // Build base image
-    NSData *data = [playback lastImg];
-    icon = (data) ? [[NSImage alloc] initWithData:data] :
-                    [NSImage imageNamed:@"missing-album"];
-    [icon setSize:size];
-    
-    // draw the overlay image (if there is album art)
-    if (data) {
-      NSImage *overlay = [NSImage imageNamed:(playback.playing.isPlaying) ?
-                          @"play" : @"pause"];
-      NSSize overlaySize = {.width = 12, .height = 12};
-      [overlay setSize:overlaySize];
-    
-      [icon lockFocus];
-      [overlay drawInRect:NSMakeRect(3, 3, [overlay size].width, [overlay size].height)
-                 fromRect:NSZeroRect
-                operation:NSCompositeSourceOver
-                 fraction:1.0];
-      [icon unlockFocus];
-    }
+    icon = [self buildPlayPauseAlbumArtImage:size];
     
   } else {
     // Use color application image
-    icon = [[NSApp applicationIconImage] copy];
+    icon = [NSImage imageNamed:@"pandora"];
```

Comment:
Yeah, so if it changes to be the album art, then it's no longer the Hermes app icon.  Does that make sense?